src/main/resources/db/changelog/changes/v1.1.0/changelog-v1.1.0.xml

```diff
@@ -106,4 +106,31 @@
     <sqlFile path="sql/create-mat-view-languages.sql" relativeToChangelogFile="true"/>
   </changeSet>
 
+  <!-- These slow change sets should be last, since they depend on previously run change sets! -->
+  <!-- Note: These don't do concurrent refreshes, since that requires the views to have been refreshed before -->
+  <changeSet id="MODFQMMGR-210-refresh-payment-status" author="mweaver@ebsco.com" runOnChange="true" context="slow">
```

Comment:
Kind of annoying to have these all in different changesets, but it makes it so if 1 fails for some reason and needs re-run, we don't have to refresh them all. It also lets us use preconditions, which is nice